pymeasure/instruments/racal/racal1992.py

```diff
@@ -0,0 +1,360 @@
+                if value < -51 or value > 51:
+                    raise ValueError(f"{value} is out of range for {setting}")
+                trigger_str = f"SL{channel_name} {value}"
+                continue
+
+            if value not in accepted_values:
+                raise ValueError(f"{value} is not an acceptable value for {setting}")
+
+            command = accepted_values[value]
+            commands.append(command)
+
+        if trigger_str != "":
+            commands.append(trigger_str)
+
+        self.write(" ".join(c for c in commands))
+
+    # ============================================================
+    # IP - Instrument Preset
+    # ============================================================
+    def preset(self):
+        """ Configure instrument with default presets."""
+        self.write('IP')
+
+    # ============================================================
+    # RE - Reset measurement
+    # ============================================================
+    def reset_measurement(self):
+        """ Reset ongoing measurement."""
+        self.write('RE')
+
+    # ============================================================
+    # Wait for measurement value
+    # ============================================================
+    def wait_for_measurement(self, timeout=None, progressDots=False):
+        """ Wait until a new measurement is available.
+
+        :param timeout: number of seconds to wait before timeout exception.
+        :param progressDots: when true, print '.' after each ready-check
+
+        """
+        if timeout is not None:
+            end_time = time.time() + timeout
+
+        while True:
+            if progressDots:
+                log.info(".")
+
+            stb = self.adapter.connection.read_stb()
+            if stb & 0x10:
+                break
+
+            if timeout is not None and time.time() > end_time:
+                raise Exception("Timeout while waiting for measurement")
+
+        return stb
+
+    # ============================================================
+    # Measured value
+    # ============================================================
+    @property
+    def measured_value(self):
+        """Measured value.
+
+        A Racal-Dana 1992 doesn't return measurement data after a request for
+        measurement data. Instead, it fills a FIFO with data whenever it completes
+        a measurement. When the FIFO is full, the oldest measurement is removed.
+
+        The FIFO buffer gets cleared when a command is received that requires
+        an immediate reply, such reading a setting. It also gets cleared when
+        an operating mode is cleared.
+
+        When there is no measurement data, this property will stall until data
+        is avaiable. It was also timeout after a time that can be set with the standard
+        pyvisa API.
+
+        One can make sure that measurement data is available by furst calling
+        `wait_for_measurement()`.
```

Comment:
```suggestion
        When there is no measurement data, this property will stall until data
        is available. It will also timeout after a time that can be set with the standard
        pyvisa API.

        One can make sure that measurement data is available by first calling
        `wait_for_measurement()`.
```